[unittest] test_get_events_grouped_by_category_limit isn't stable

this test keep failing on some timeout, almost on each PR that being open (at least once)

it's wasting our time, by looking into it, and by retrying

```
FAILED unit_tests/test_sct_events_file_logger.py::TestFileLogger::test_get_events_grouped_by_category_limit - AssertionError: Events process EventsFileLogger is still alive after timeout
```

if we can find the root cause quickly let's fix,
if not, let's skip it for now (or move it to integration marker)

recent example of it:
https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-6746/2/execution/node/36/log/

--- a/unit_tests/test_sct_events_file_logger.py
+++ b/unit_tests/test_sct_events_file_logger.py
@@ -38,7 +38,7 @@ class TestFileLogger(unittest.TestCase, EventsUtilsMixin):
         self.assertEqual(self.file_logger._registry, self.events_processes_registry)
 
     def tearDown(self) -> None:
-        self.file_logger.stop(timeout=1)
+        self.file_logger.stop(timeout=3)
         self.teardown_events_processes()
 
     def test_file_logger(self) -> None:

